app/controllers/account_controller.rb

```diff
@@ -1,11 +1,11 @@
-class ProfileController < ApplicationController
+class AccountController < ApplicationController
```

Comment:
rails controllers are usually plural -- thoughts on calling this `AccountsController` ?